k8s/k8s.go

```diff
@@ -66,19 +67,24 @@ func ApplyDeployment(path string, mutators ...func(deployment *v1.Deployment)) e
 	if err != nil {
 		return err
 	}
-	var d v1.Deployment
+	var d apiv1.Deployment
 	if parseErr := yaml.Unmarshal(b, &d); parseErr != nil {
 		return parseErr
 	}
 	for _, m := range mutators {
 		m(&d)
 	}
-	_, err = client.AppsV1().Deployments(namespace).Create(&d)
+
+	if d.Namespace == "" {
+		d.Namespace = namespace
```

Comment:
Why do we need this?

Reply:
We can't use empty namespaces in k8s api calls